xml/art_sles_rpi_quick.xml

```diff
@@ -376,8 +379,8 @@
     <para>
      For other limitations refer to the online version of the Release Notes at
      <link
-            xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15/"
-            >https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15/</link>.
+            xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP1/"
```

Comment:
URLs must not have labels, because they do not print correctly in PDF format. Use a plain URL, 
&lt;link xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP1/"/>

Reply:
FTR I did test the pdf version and did not notice a problem. Your proposal does not look different but is a simplification, so I'll convert all RPi links in one go as a separate commit to distinguish from the URL update.